index.html

```diff
@@ -19,7 +19,7 @@
             <a class="anchor" href="#">Cart Management</a>
         </li>
         <li class="pages-list">
-            <a class="anchor" href="">Wishlist Management</a>
+            <a class="anchor" href="./screens/wistList/wishlist.html">Wishlist Management</a>
```

Comment:
On home page, nav links are not visible properly, please check the contrast ratio

Reply:
done